components/Table/TableKDA.jsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import uuid from 'node-uuid';
+import ReactTooltip from 'react-tooltip';
+import styles from './TableKDA.css';
+
+const TableKDA = ({ kills, deaths, assists, matchId = uuid.v4() }) => {
+  const kdaSum = kills + deaths + assists;
+
+  return (
+    <div className={styles.container}>
+      <div>
+        {kills}
+      </div>
+      <div>
+        <div className={styles.percent} data-tip data-for={`kda-${matchId}`}>
+          <div style={{ width: `${(kills * 100) / kdaSum}%`, backgroundColor: styles.red }} />
+          <div style={{ width: `${(deaths * 100) / kdaSum}%`, backgroundColor: styles.neutral }} />
+          <div style={{ width: `${(assists * 100) / kdaSum}%`, backgroundColor: styles.green }} />
+        </div>
+        <ReactTooltip id={`kda-${matchId}`} place="right" type="light" effect="float">
+          {`KDA: ${Math.round((kdaSum / deaths) * 100) / 100}`}
```

Comment:
I made a mistake with kda here.
The right way is `Number(((kills + assists) / (!deaths ? 1 : deaths)).toFixed(2))`